presto-product-tests/src/main/java/com/facebook/presto/tests/cassandra/MultiColumnKeyTableDefinition.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.tests.cassandra;
+
+import com.google.common.collect.ImmutableList;
+import com.teradata.tempto.fulfillment.table.jdbc.RelationalDataSource;
+import com.teradata.tempto.internal.fulfillment.table.cassandra.CassandraTableDefinition;
+
+import java.sql.JDBCType;
+import java.sql.Timestamp;
+import java.util.List;
+
+import static com.facebook.presto.tests.cassandra.TestConstants.CONNECTOR_NAME;
+import static com.facebook.presto.tests.cassandra.TestConstants.KEY_SPACE;
+import static java.sql.JDBCType.TIMESTAMP;
+import static java.sql.JDBCType.VARCHAR;
+
+public class MultiColumnKeyTableDefinition
+{
+    private MultiColumnKeyTableDefinition() {}
+
+    public static final ImmutableList<JDBCType> MULTI_COLUMN_KEY_TYPES = ImmutableList.of(VARCHAR, VARCHAR, VARCHAR, TIMESTAMP);
+    private static final String MULTI_COLUMN_KEY_DDL =
+            "CREATE TABLE %NAME% (" +
+                    "user_id text, " +
+                    "key text, " +
+                    "updated_at timestamp, " +
+                    "value text, " +
+                    "PRIMARY KEY (user_id, key));";
```

Comment:
How about adding one more column in key (having partitioning key = { user_id } and compound clustering key, e.g. { key, updated_at })?
This would enable us to test filtering by a prefix of key (which is different path use case than scanning by entire clustering key).
Then we should also test scanning on a column which is part of clustering key but is not a prefix -> Cassandra cannot help with scanning here.